frontend/src/constants/API.tsx

```diff
@@ -29,12 +29,17 @@ export interface IParcelListParams {
   maxLandArea: number | null;
 }
 export const PARCELS = (params: IParcelListParams | null) =>
-  params ? `/parcels?${queryString.stringify(params)}` : '/parcels'; // get filtered properties or all if not specified.
+  params ? `/properties?${queryString.stringify(params)}` : '/properties'; // get filtered properties or all if not specified.
 export interface IParcelDetailParams {
   id: number;
 }
 export const PARCEL_DETAIL = (params: IParcelDetailParams) => `/parcels/${params.id}`;
 
+export interface IBuildingDetailParams {
```

Comment:
The logic for calling the `/buildings/{id}` endpoint needs to be added to the map.